docs/source/basic_tutorials/consuming_tgi.md

```diff
@@ -0,0 +1,26 @@
+# Consuming Text Generation Inference
```

Comment:
Add a brief sentence or two here introducing this doc. Something like:

TGI powers several of the products at Hugging Face. You can use it with [Chat UI](https://github.com/huggingface/chat-ui), an open-source chat interface for LLMs to build your own chatbot, or with the [`~huggingface_hub.InferenceClient`] to perform inference.

Reply:
Addressed this.